### PR TITLE
Bring back hemophilia. No, it's not a crime!

### PR DESCRIPTION
**Pull request by lbnesquik**
It is a negative trait that makes you bleed a loss.
It comes with a hacked together system to deny synths from taking it. It was a PAIN.

### CODE REVIEW

File: code/modules/client/preference_setup/vore/07_traits.dm [hunk 1]
```diff
@@ -255,6 +260,14 @@
 
 			var/conflict = FALSE
 
+			user.isSynthetic()	//Recheck just to be sure
```
Comment (ShadowLarkens) [inline code/modules/client/preference_setup/vore/07_traits.dm:263]: Is this meant to be `pref.dirty_synth = user.isSynthetic()`?

Comment (lbnesquik) [reply code/modules/client/preference_setup/vore/07_traits.dm:263]: 🤔 That wasn't what i was trying to do, but it might work just as well if not better.

Comment (lbnesquik) [reply code/modules/client/preference_setup/vore/07_traits.dm:263]: Tested, didn't seem to quite work.

Comment (ShadowLarkens) [reply code/modules/client/preference_setup/vore/07_traits.dm:263]: Hrm, it just seems like this line doesn't actually do *anything* right now, I don't think isSynthetic() has any side effects?

**Comment by KasparoVy** [comment]
Blood for the blood god

**Comment by lbnesquik** [comment]
Staff discussion required?

**Comment by nerdass** [comment]
what happens if someone selects the trait and then picks fullbody prosthetic in character setup?

**Comment by KasparoVy** [comment]
They bleed to death, white synth blood everywhere!!! Aaaaaaa

**Comment by lbnesquik** [comment]
> 
> 
> what happens if someone selects the trait and then picks fullbody prosthetic in character setup?

That's why there is the synth checks

**Comment by lm40** [comment]
provided the code is solid this seems good

**Comment by SilencedMP5A5** [comment]
yee

**Comment by nerdass** [comment]
no i meant like, AFTER they select the trait.

**Comment by MrSign** [comment]
Good staff discussion. Very insightful. MERGE!

**Comment by KasparoVy** [comment]
Linkin Park - Bleed it Out lmao

**Pull request merged**